awscli/customizations/s3/subcommands.py

```diff
@@ -540,8 +541,11 @@ def _handle_rm_force(self, parsed_globals, parameters):
         # to be kept simple.
         if 'force' in parameters:
             if parameters['force']:
-                bucket = find_bucket_key(parameters['src'][5:])[0]
-                path = 's3://' + bucket
+                bucket = find_bucket_key(parameters['src'][5:])
+                if bucket[1]:
```

Comment:
@kyleknap what do you think about line this line? The ```find_bucket_key``` function always returns an array with two elements, with the second element sometimes being empty. This looks like it may throw an ```IndexError``` by it's really just checking to see if the second element is an empty string.